invenio_app_ils/mail/messages.py

```diff
@@ -50,7 +56,13 @@ def __init__(
             )
         )
         self.ctx = ctx
-
+        footer_template = dict(
```

Comment:
Maybe better to keep it a string?
```suggestion
        footer_template = current_app.config.get("ILS_GLOBAL_MAIL_TEMPLATES", {})["footer"] or self.default_footer_template
```